JSDemos/Demos/TreeList/ColumnFixing/description.md

```diff
@@ -1 +1,5 @@
-If&nbsp;the TreeList contains many columns, they may exceed the width of&nbsp;the component and cause horizontal scrolling. At&nbsp;the same time, some pivotal columns should be&nbsp;on&nbsp;screen no&nbsp;matter how far the TreeList is&nbsp;scrolled. Such columns can be&nbsp;fixed to&nbsp;the right or&nbsp;to&nbsp;the left of&nbsp;the TreeList using the context menu appearing when a&nbsp;user right-clicks a&nbsp;column header.
\ No newline at end of file
+When the width of all columns exceeds the component's width, horizontal scrolling occurs. TreeList allows to fix certain columns to the right or left so they would always be visible on scroll. To enable this feature, set **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) to **true**.
+
+To fix a column programmatically, set its [fixed](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixed) property to **true**. The default position is TreeList's left, but you can set the column's [fixedPosition](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#fixedPosition) property to *"right"*.
+
+To fix/unfix a column in the UI, users should right-click the column's header to open the context menu and choose the desired operation. If some column should never be fixed or unfixed, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) to **false**.
```

Comment:
We are talking about UI in the first paragraph. It would be logical to move this paragraph up.

```suggestion
To fix/unfix a column, users should right-click the column's header and select the desired operation in the appeared context menu. If a column should never be fixed or unfixed, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.
```